.gitignore

```diff
@@ -1 +1,2 @@
 .data
+.vscode/**
```

Comment:
might be better to put this in your global .gitignore, but doesn't really hurt anyone having it here